scripts/process_spraggins.sh

```diff
@@ -8,64 +8,70 @@ main() {
     # mass spectrometry (IMS) experiment. A single zarr store is created.
 
     get_CLI_args "$@"
-    IMZML_IN="$INPUT/spraggins.ims.imzml"
-    IBD_IN="$INPUT/spraggins.ims.ibd"
-    ZARR_OUT="$OUTPUT/spraggins.ims.zarr"
-    JSON_OUT="$OUTPUT/spraggins.ims.json"
 
+    # IMS data
+    IMS_IMZML_IN="$INPUT/spraggins.ims.imzml"
+    IMS_IBD_IN="$INPUT/spraggins.ims.ibd"
+    IMS_ZARR_OUT="$OUTPUT/spraggins.ims.zarr"
+    IMS_JSON_OUT="$OUTPUT/spraggins.ims.raster.json"
+    IMS_RASTER_NAME="Spraggins IMS"
+
+    # MXIF data
+    MXIF_TIFF_IN="$INPUT/spraggins.mxif.ome.tif"
+    MXIF_ZARR_OUT="$OUTPUT/spraggins.mxif.zarr"
+    MXIF_JSON_OUT="$OUTPUT/spraggins.mxif.raster.json"
+    MXIF_RASTER_NAME="Spraggins MxIF"
+
+
+    # CLOUD SOURCE
+    SOURCE_URL="https://vitessce-data.s3.amazonaws.com/source-data/spraggins"
+
+    # CLOUD TARGET
     RELEASE=${CLOUD_TARGET//vitessce-data\//}
-    ZARR_STORE_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/spraggins"
+    SERVER_URL="https://vitessce-data.storage.googleapis.com/$RELEASE/spraggins/"
```

Comment:
If there is `SOURCE_URL`, maybe this should be `DEST_URL`? `SERVER` doesn't tell us much.

Reply:
Though I see it matches the command-line argument... Whatever you think best.